Stack/Opc.Ua.Core/Security/Certificates/CertificateValidator.cs

```diff
@@ -696,7 +698,8 @@ private void SaveCertificates(X509Certificate2Collection certificateChain)
                             {
                                 try
                                 {
-                                    store.Add(certificate).GetAwaiter().GetResult();
+                                    // number of certs for history + current chain size
+                                    store.AddRejected(certificate, certificateChain.Count + kMaxRejectedCertificates).GetAwaiter().GetResult();
```

Comment:
Doesn't this call break existing custom rejected store implementations?

Reply:
yes, intentionally only support rejected store on file systems.